temporian/implementation/numpy/operators/glue.py

```diff
@@ -37,9 +37,7 @@ def __call__(
         output_schema = self.output_schema("output")
 
         # convert input evest dict to list of evsets
-        evsets: List[EventSet] = list(
-            list(zip(*sorted(list(inputs.items()))))[1]
-        )
+        evsets: List[EventSet] = list(list(zip(*list(inputs.items())))[1])
```

Comment:
This is confusing. If we don't need to sort, can we just do `list(inputs.values())` ?

Reply:
lol. thanks. updated.